srcpkgs/python3-chardet/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'python3-chardet'
 pkgname=python3-chardet
 version=4.0.0
-revision=3
+revision=4
 wrksrc="chardet-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
-depends="python3-setuptools"
+depends="python3"
 checkdepends="python3-pytest"
 short_desc="Universal encoding detector"
 maintainer="Alessio Sergi <al3hex@gmail.com>"
 license="LGPL-2.1-only"
 homepage="https://github.com/chardet/chardet"
 distfiles="${PYPI_SITE}/c/chardet/chardet-${version}.tar.gz"
 checksum=0d6f53a15db4120f2b08c94f11e7d93d2c911ee118b6b30a04ec3ee8310179fa
-alternatives="chardet:chardetect:/usr/bin/chardetect2"
+conflicts="python-chardet"
```

Comment:
i don't think this is necessary, the py2 version is a removed package

Reply:
True, but it still exists in the repos and can be installed (or someone could have it installed and ignored), and as we all know the `alternatives` do have some issues. So, isn't it better to keep it around until the pkg is removed from the repos as well?

Reply:
We have been converting alternatives to conflicts when dropping py2 packages to provide some explicit indicator for people who might keep the py2 package around. This is consistent with broader practice.